style.json

```diff
@@ -1,25 +1,24 @@
 {
   "version": 8,
-  "name": "Fluds",
+  "name": "Floods",
   "metadata": {
-    "maputnik:license": "https://github.com/maputnik/osm-liberty/blob/gh-pages/LICENSE.md",
     "maputnik:renderer": "mbgljs",
     "openmaptiles:version": "3.x"
   },
   "sources": {
-    "fludpoints": {
+    "floodpoints": {
       "type": "geojson",
       "data": "https://raw.githack.com/HarelM/IHMMapboxMapExample/main/source.geojson"
     }
   },
-  "sprite": "https://maputnik.github.io/osm-liberty/sprites/osm-liberty",
+  "sprite": "",
   "glyphs": "https://api.maptiler.com/fonts/{fontstack}/{range}.pbf?key={key}",
```

Comment:
Probably better to change this as well, while you are at it...

Reply:
Change what to what?

Reply:
Orangemug?
Forget this is also ignored.
Maybe we can use our fonts...?

Reply:
Feel free to modify this PR as you like.